add shared responsibility model page (#380)
aziontech/docs: src/content/docs/en/pages/shared-responsibility/shared-responsibility.mdx

---
title: Shared Responsibility Model
description: This document highlights the roles and responsibilities of each party when organizations use the Azion Edge Platform.

> meta_tags: azion, shared responsibility, 

Reviewer: ```suggestion
meta_tags: azion, shared responsibility
```

namespace: docs_shared_responsibility
permalink: /documentation/shared-responsibility/
---

This document highlights the roles and responsibilities of each party when organizations use the Azion Edge Platform. Security and compliance are a shared responsibility between customers and Azion. Azion is responsible for the security and compliance of its Edge Platform, and customers are responsible for the content, their application, and the correct operation of the provided controls.

Customers are advised to carefully select Azion’s Products and Services and operate them according to the applicable compliance requirements, laws, and regulations.

---

## Cloud Delivery Models

NIST Special Publication 500-292 defines five essential characteristics of cloud services:

- On demand self-service 
- Broad network access 
- Resource pooling 
- Rapid elasticity 
- Measured service

> The publication also provides a simple and unambiguous categorisation of cloud services models:

Reviewer 2: ```suggestion
The publication also provides a simple and unambiguous categorization of cloud services models:
```

- Cloud software as a service (SaaS)
- Cloud platform as a service (PaaS)
- Cloud infrastructure as a service (IaaS)

> Azion’s Edge Platform does not accurately fit into any of the three delivery models and can be described as a blend of PaaS and SaaS models.

Reviewer 3: ```suggestion
Azion’s Edge Platform doesn't accurately fit into any of the three delivery models and can be described as a blend of PaaS and SaaS models.
```

---

## Shared Responsibility Model

> The shared responsibility model is central to understanding the roles of the customers and Azion in the context of application security and the use of Azion Edge Platform.

Reviewer 3: ```suggestion
The Shared Responsibility Model is central to understanding the roles of the customers and Azion in the context of application security and the use of Azion Edge Platform.
```

> Customers are responsible for the correct configuration and usage of the controls and for the application and content they put on or through Azion’s Edge Platform.

Reviewer 3: ```suggestion
Customers are responsible for the correct configuration and usage of the controls and for the application and content they put on or through Azion Edge Platform.
```

Customers can use Azion compliance documents to perform any required Azion control evaluations.

### Customer's Responsibilities

“Security in the Edge Platform” – Customer responsibility will be determined by the Azion Product and Service that a customer selects. This determines the amount of configuration work the customer must perform as part of their security responsibilities.

For abstracted Products and Services on a as-a-service or serverless model, which uses Azion Global Network, Azion develops, operates, protects, and controls the infrastructure, while customers access the Products and Service interfaces to store, manage, and retrieve data. Customers are responsible for items such as:

- Correct configuration of Azion Products and Services.
- Configuration change control.
- Managing their data, such as content and application and source code (including encryption options).
- Identity and access management, including user provisioning and their roles.
- Monitoring quotas and limits.
- Monitoring consumption and billing.
- Correct usage of Azion’s platform.

Customers are also responsible for:

> - Adhering to Azion’s Acceptable Use Policy and all applicable laws and regulations.

Reviewer 3: ```suggestion
- Adhering to [Azion’s Acceptable Use Policy](/en/documentation/agreements/acceptable-use-policy/) and all applicable laws and regulations.
```
